en/docs/install-and-setup/setup/user-stores/configure-sso-userstore-in-icp.md

```
@@ -0,0 +1,296 @@
+    <tr>
+      <td><code>ssoScopes</code></td>
+      <td>OIDC scopes to request. <code>openid</code> is required.</td>
+    </tr>
+  </tbody>
+</table>
+
+## Provider-specific examples
+
+### Asgardeo
+
+```toml
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)